bravado/fido_client.py

```diff
@@ -60,7 +60,7 @@ def text(self):
 
     @property
     def raw_bytes(self):
-        # type: () -> bytes
+        # type: () -> typing.Union[bytes, str]
```

Comment:
This should always be bytes I thought - can you tell me more about this?

Reply:
We did check it offline.

The reason for which I've added `Union[bytes, str]` was that:
 * on python2 `self._delegate.body` (for fido) and `self._delegate.content` (for requests) are returning `str`
 * on python3 `self._delegate.body` (for fido) and `self._delegate.content` (for requests) are returning `bytes`

But on python2 `str == bytes` so the `Union` is not needed.

I'm reverting the change